includes/wikia/WikiaSQLIterator.class.php

```diff
@@ -0,0 +1,139 @@
+<?php
+/**
+ * WikiaSQLIterator
```

Comment:
Great work!
And I think it would be helpful (in distant future) to put here some description - what for is that module, and why it might be useful. Something similar like this PR description - even if you copy and paste most of it, along with the example :)